dot/state/network.go

```diff
@@ -133,3 +133,20 @@ func (ns *NetworkState) SetPeers(peers *[]common.PeerInfo) error {
 	err = ns.db.Put(peersKey, enc)
 	return err
 }
+
+// GetRequestedBlock retrieves network state from the database
+func (ns *NetworkState) GetRequestedBlock(blockId string) (bool, error) {
```

Comment:
this isn't something we need to save in the database, it's fine if it gets cleared out upon restarts. what do you think?